packages/taquito-local-forging/test/codec.spec.ts

```diff
@@ -39,4 +48,144 @@ describe('Entrypoint', () => {
       })
     );
   });
+
+  test(`Verify pkhEncoder`, async (done) => {
+    const tz1 = pkhEncoder('tz1e42w8ZaGAbM3gucbBy8iRypdbnqUj7oWY');
+    expect(tz1).toEqual('00c9fc72e8491bd2973e196f04ec6918ad5bcee22d');
+    const tz2 = pkhEncoder('tz2Ch1abG7FNiibmV26Uzgdsnfni9XGrk5wD');
+    expect(tz2).toEqual('012ffebbf1560632ca767bc960ccdb84669d284c2c');
+    const tz3 = pkhEncoder('tz3WXYtyDUNL91qfiCJtVUX746QpNv5i5ve5');
+    expect(tz3).toEqual('026fde46af0356a0476dae4e4600172dc9309b3aa4');
+    expect(() => pkhEncoder('tz4WXYtyDUNL91qfiCJtVUX746QpNv5i5ve5')).toThrow(InvalidKeyHashError);
+    expect.objectContaining({
```

Comment:
Missing something, there are other occurrences in the file that needs to be fixed

Reply:
fixed all of the incorrecty written tests of the exception message thrown